lookout/style/typos/cmdline_tools.py

```diff
@@ -0,0 +1,291 @@
+"""
+Command line utilities to check the quality of a model on a given dataset, visualize errors, etc.
+"""
+from argparse import ArgumentParser
+from typing import Any
+
+from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.core.slogging import setup as setup_slogging
+
+from lookout.style.typos.train_test_corrector import (DEFAULT_FREQUENCY_PATH,
+                                                      DEFAULT_VOCABULARY_PATH,
+                                                      DEFAULT_VOCABULARY_SIZE)
+
+
+def add_input_path_arg(my_parser: ArgumentParser, addition: str = ""):
+    """
+    Add an input path argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    :param addition: Addition to the default help message.
+    """
+    my_parser.add_argument(
+        "-i", "--input-path", required=False, type=str,
+        help="Path to a .csv dump of input dataframe. Should contain column 'token_split" +
+             addition)
+
+
+def add_frequency_column_arg(my_parser: ArgumentParser):
+    """
+    Add a frequency column argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-fc", "--frequency_column", required=False, type=str,
+        help="Name of column with identifiers frequencies. If not specified,"
+             "every split is considered to have frequency 1.")
+
+
+def add_vocabulary_size_arg(my_parser: ArgumentParser):
+    """
+    Add a vocabulary size argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-vs", "--vocabulary_size", required=False, type=int,
+        default=DEFAULT_VOCABULARY_SIZE,
+        help="Number of most frequent tokens to take as a vocabulary.")
+
+
+def add_frequencies_size_arg(my_parser: ArgumentParser):
+    """
+    Add a frequencies size argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-fs", "--frequencies_size", required=False, type=int,
+        help="Number of most frequent tokens to save frequencies info for."
+             "If not specified, frequencies for all present tokens will be saved.")
+
+
+def add_vocabulary_path_arg(my_parser: ArgumentParser):
+    """
+    Add a vocabulary path argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-vp", "--vocabulary_path", required=False, type=str,
```

Comment:
```suggestion
        "-vp", "--vocabulary-path", required=False, type=str,
```
same here `-vp`. Probably we should exclude short options completely: there are too many.